README.md

```diff
@@ -6,9 +6,7 @@ This is the development repo for mu-plugins on [VIP Go](https://wpvip.com/docume
 
 ### Local Dev
 
-We recommend using the Lando-based development environment for local development: https://github.com/Automattic/vip-go-mu-dev
-
-Follow the instructions in the `vip-go-mu-dev` repo to get set up (it includes a clone of this repo).
+We recommend using the VIP local development environment for local development: https://docs.wpvip.com/technical-references/vip-local-development-environment/
```

Comment:
Since our usage of the environemtn will be slightly different than customers, it would be better to include specific details here around how to set it up, including mounts and such. Even better if we can just have a `bin` script that we can run, similar to what I shared in Slack:

```
mkdir dev
cd dev
git clone git@github.com:WordPress/WordPress.git wordpress
git clone https://github.com/Automattic/vip-go-skeleton skeleton
git clone git@github.com:Automattic/vip-go-mu-plugins.git mu-plugins
vip dev-env create --slug vip-dev --title "VIP Dev" --wordpress ./wordpress --client-code ./skeleton --mu-plugins ./mu-plugins
vip dev-env start --slug vip-dev
```

Bonus: have the script / snippet set up a single site and multisite environment.

Reply:
Good point. I added a little bit more instructions there.

However, I opted for a more simple example than what you suggested. For two reasons:
* The `vip` cli has the creation wizard that walks you through the options, so we don't need to document their existence here
* I don't think having a local skeleton, wordpress is that needed for a user unfamiliar with mu-plugins development, which is the target audience here. So making a spinning up dev environment simpler, without the extra steps seems like a good tradeoff to me.

I am happy to hear contra arguments though.